pkg/api/v1/types.go

```diff
@@ -1961,13 +1961,30 @@ type PodLogOptions struct {
 	// The container for which to stream logs. Defaults to only container if there is one container in the pod.
 	Container string `json:"container,omitempty"`
 
-	// Follow the log stream of the pod.
-	// Defaults to false.
+	// Follow the log stream of the pod. Defaults to false.
 	Follow bool `json:"follow,omitempty"`
 
-	// Return previous terminated container logs.
-	// Defaults to false.
+	// Return previous terminated container logs. Defaults to false.
 	Previous bool `json:"previous,omitempty"`
+
+	// An RFC3339 timestamp or a relative time duration relative to the current time
+	// from which to show logs. If this value precedes the time a pod was started, only
+	// logs since the pod start will be returned. If this value is in the future, no logs
+	// will be returned.
+	Since string `json:"since,omitempty"`
```

Comment:
I would like to preserve using relative time though with duration, instead of requiring use of seconds.  As a client or CLI user it's a lot more convenient.  It is however more *magic* than our current API design.